playbooks/openshift-node/private/join.yml

```diff
@@ -16,10 +17,35 @@
     - hostvars[item].openshift.common is defined
     - (hostvars[item].openshift_is_bootstrapped | bool) or (hostvars[item].openshift_node_bootstrap | default(True) | bool)
     changed_when: False
+  - name: Create a tempdir for node configs
+    command: mktemp -d /tmp/openshift-ansible-XXXXXXX
+    register: mktemp
```

Comment:
I'm not sure this will persist across plays.  Probably need to use set_fact, but not 100% sure.

Reply:
There is no need to persist those - the source of node configs is master anyway